frontend/src/components/GridWasteMetrics.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import WasteRow from './WasteRow'
+export default function GridWasteMetrics() {
+  return (
+    <div className="grid-metrics">
+        <div className="metrics-title">Waste Metrics</div>
+        <div className ="metrics-cells-container">
+              <div className="row-element">
+                  <div>Name</div>
```

Comment:
These should be spans, rather than divs.

Reply:
Also, if you are making a table, you _can_ use a table, tr and td tags. Tailwind and the UI libraries should make sure that the formatting is consistent. Semantic tags are useful for accessibility purposes.

Reply:
table has been implemented. Just need to work on styling it, and fixing the head of the table. Changes have been pushed to reflect that.